doc/mgr/dashboard.rst

```diff
@@ -604,6 +604,43 @@ dashboard in the future.
 User and Role Management
 ------------------------
 
+Password Policy
+^^^^^^^^^^^^^^^
+
+By default the password policy feature is enabled including the following checks:
+
+- Is the password longer than N characters?
+- Are the old and new password the same?
+
+The password policy feature can be switched on or off completely::
+
+    $ ceph dashboard set-pwd-policy-enabled <true|false>
+
+Individual tests can be switched on or off::
+
+  $ ceph dashboard set-pwd-policy-check-length-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-oldpwd-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-username-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-exclusion-list-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-complexity-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-sequential-chars-enabled <true|false>
+  $ ceph dashboard set-pwd-policy-check-repetitive-chars-enabled <true|false>
+
+Additionally the following options are available to configure the password policy behaviour.
+
+- The minimum password length (defaults to 8)::
+
+  $ ceph dashboard set-pwd-policy-min-length <N>
+
+- The minimum password complexity (defaults to 10)::
```

Comment:
Is this complexity level explained somewhere? What is the minimum and maximum value? Which enforces more complexity? What does "complexity" actually mean here?

Reply:
> Is this complexity level explained somewhere? What is the minimum and maximum value? Which enforces more complexity? What does "complexity" actually mean here?

No, the origin author of this function has not added any documentation or reference. Seems i had to go through the code try to extract the rules from there.